thriftpy2/parser/__init__.py

```diff
@@ -39,6 +39,9 @@ def load(path,
         fill_incomplete_ttype(thrift, thrift)
     if real_module:
         sys.modules[module_name] = thrift
+        for module in thrift.__thrift_meta__["includes"]:
```

Comment:
Do we need add these modules recursively?

Reply:
Sure, if the current solution if what we want，there should rewrite to recursion.